lib/macro/add_contract_error.rb

```diff
@@ -1,15 +1,21 @@
 # frozen_string_literal: true
 
 module Macro
-  def self.AddContractError(name: :default, **args)
-    task = Trailblazer::Activity::TaskBuilder::Binary(
-      ->(ctx, **) {
-        error, message = args.first
-        ctx["contract.#{name}"].errors.add(
-          error, message.is_a?(Array) ? message : I18n.t(message)
-        )
+  class AddContractError
```

Comment:
Not sure that it's a good idea. Now we always should create a new instance, instead of calling the method directly.